script.js

```diff
@@ -6,7 +6,7 @@ if (window.matchMedia("(max-width: 500px)").matches) {
     picture_canvas.width = width;
     picture_canvas.height = height;
 } else {
-    var width = 640;
+    var width = 640; // delete these if incoming video dimensions are overwriting them? 
```

Comment:
is this comment a todo?

Reply:
no, i modified the comment to "default values, overwritten by incoming video dimensions"